src/routes/changelog/(entries)/2025-03-24.markdoc

```diff
@@ -0,0 +1,26 @@
+---
+layout: changelog
+title: Appwrite Cloud backups now faster
+date: 2025-03-24
+---
+
+Appwrite Cloud backups are now considerably faster, reducing wait times and improving reliability.
+
+## What's changed
+
+We made key changes to the backup infrastructure by moving data reading operations inside the network, and implementing I/O optimizations for storage operations.
+
+Internal testing revealed notable time savings across database backups:
+
+- **3 minute backup → 1.5 minutes** (2x faster)
+- **3.5 minute backup → 1.2 minutes** (3x faster)
+- **6.7 minute backup → 2 minutes** (3.4x faster)
+- **25.8 minute backup → 8.7 minutes** (3x faster)
+
+Most databases showed at least a 2x reduction in backup time, with many achieving up to 7x faster backups depending on size and structure.
+
+These optimizations improve the efficiency of data recovery and management. Your actual performance gains may vary depending on database size and structure.
```

Comment:
"Depending on size and structure" is repeated in these two blocks, let's remove from the first

Reply:
Sure, done